pom.xml

```diff
@@ -724,14 +724,14 @@
 					</configuration>
 				</plugin>
 				<plugin>
-					<groupId>org.codehaus.sonar</groupId>
+					<groupId>org.sonarsource.scanner.maven</groupId>
 					<artifactId>sonar-maven-plugin</artifactId>
-					<version>4.5.4</version>
```

Comment:
Is this a version downgrade? 😊

Reply:
Not really. The plugin changed, as far as I can understand. But then I'm just following official docs.